testdata/baselines/reference/submodule/conformance/templateLiteralTypesPatterns.errors.txt.diff

```diff
@@ -81,27 +77,15 @@
      
      declare function numbers(x: `${number}`): void;
      // the following should work
-@@= skipped -71, +71 lines =@@
-     declare function bigints(x: `${bigint}`): void;
-     // the following should work
-     bigints("1");
-+            ~~~
-+!!! error TS2345: Argument of type '"1"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("-1");
-+            ~~~~
-+!!! error TS2345: Argument of type '"-1"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("0");
-+            ~~~
-+!!! error TS2345: Argument of type '"0"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("0b1");
-+            ~~~~~
-+!!! error TS2345: Argument of type '"0b1"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("0x1");
-+            ~~~~~
-+!!! error TS2345: Argument of type '"0x1"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("0o1");
-+            ~~~~~
-+!!! error TS2345: Argument of type '"0o1"' is not assignable to parameter of type '`${bigint}`'.
+@@= skipped -133, +133 lines =@@
+     
+     // whitespace and comments aren't part of numbers
+     bigints("- 1");
+-            ~~~~~
+-!!! error TS2345: Argument of type '"- 1"' is not assignable to parameter of type '`${bigint}`'.
```

Comment:
This is missing; will investigate to figure out where this goes wrong.

Reply:
It turns out our scanner doesn't respect skipTrivia in all of the places it's supposed to.